pkg/distributor/push_test.go

```diff
@@ -846,6 +845,114 @@ func TestHandler_ErrorTranslation(t *testing.T) {
 		})
 	}
 }
+func TestHandler_HandleRetryAfterHeader(t *testing.T) {
+	const (
+		DefaultRetryBase      = 3 * time.Second
+		DefaultMaxAllowedAtts = 2
```

Comment:
nitpick: have these two start with lower-case. Link to the [same style guide](https://google.github.io/styleguide/go/decisions.html#constant-names) talking about constants